qt/widgets/common/test/FitScriptGeneratorViewTest.h

```diff
@@ -172,13 +198,33 @@ class FitScriptGeneratorViewTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(selectedIndices[0].value, rowIndex);
   }
 
-  void
-  test_that_selectedRows_will_return_an_empty_vector_if_there_are_no_currently_selected_rows() {
+  void test_that_selectedRows_will_return_the_first_row_index_by_default() {
     openFitScriptGeneratorWidget();
     m_view->addWorkspaceDomain(m_wsName, m_wsIndex, 0.0, 2.0);
+    m_view->addWorkspaceDomain("Name2", m_wsIndex, 0.0, 2.0);
 
     auto const selectedIndices = m_view->selectedRows();
-    TS_ASSERT_EQUALS(selectedIndices.size(), 0);
+    TS_ASSERT_EQUALS(selectedIndices.size(), 1);
+    TS_ASSERT_EQUALS(selectedIndices[0], FitDomainIndex(0));
+  }
+
+  void
+  test_that_parameterValue_will_return_the_correct_value_of_the_specified_parameter() {
+    openFitScriptGeneratorWidget();
+    m_view->addWorkspaceDomain(m_wsName, m_wsIndex, 0.0, 2.0);
+    m_view->setFunction(m_function);
+
+    TS_ASSERT_EQUALS(m_view->parameterValue("f0.A0"), 0.0);
+    TS_ASSERT_EQUALS(m_view->parameterValue("f1.Height"), 1.0);
```

Comment:
does it still work for a flat background and a linear background? They both have an `A0`